BrainPortal/public/javascripts/cbrain/users/clipboard.js

```diff
@@ -0,0 +1,27 @@
+
+/*
+#
+# CBRAIN Project
+#
+# Copyright (C) 2024
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+*/
+
+function copy_text_to_clipboard() {
```

Comment:
This file is adding a function in the top-level javascript namespace?!? This is wrong, it becomes a global identifier.

Please add the function to cbrain.js were we isolate all the functions properly. Use a class to bind it to your element.